pom.xml

```diff
@@ -11,14 +11,14 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>angular-rickshaw</artifactId>
-    <version>0.3.1-SNAPSHOT</version>
+    <version>0.10.0</version>
```

Comment:
OK. I didn't really know about that. Is there already something prominent in the documentation that I didn't see?